BGL/test/BGL/test_cgal_bgl_named_params.cpp

```diff
@@ -88,6 +88,7 @@ void test(const NamedParameters& np)
   assert(get_param(np, CGAL::internal_np::require_same_orientation).v == 49);
   assert(get_param(np, CGAL::internal_np::use_bool_op_to_clip_surface).v == 50);
   assert(get_param(np, CGAL::internal_np::face_size_map).v == 52);
+  assert(get_param(np, CGAL::internal_np::maximum_number_of_faces).v == 58);
```

Comment:
Should not you all use random numbers instead of consecutive ones? That would avoid conflicts.

Reply:
Actually this test was introduced before the introduction of the `internal_np` namespace and is somehow useless now (because developer should always use the variable in the namespace and no confusion with the free function address should now be done).  @MaelRL do you have a different opinion?

Reply:
The test would actually have been useful once for a broken named parameter added a little while ago, but unfortunately the named parameter hadn't been added to this test and we noticed the bug somewhere else... So it's not completely useless and the time to fix the (very few) conflicts it generates is negligible so I would just keep it.

Reply:
As I said I think the issue was because we were using the function instead of the variable. Actually having variable and function names generated using a fixed pattern would be better than this test. I'll fix that when I'll be doing the np replacement.